docs/modules/ROOT/pages/index.adoc

```diff
@@ -7,3 +7,9 @@ chart] on a Syn-enabled cluster.
 The component defaults to provisioning a Bitnami Postgres database via the
 Keycloak helm chart, but this behavior can be overridden using the
 `keycloak.postgres` parameters exposed by the component.
+
+[WARNING]
+====
+This component currently doesn't support K8s versions older than 1.14.
+This is because it generates `Ingress` resources with `apiVersion: networking.k8s.io/v1`.
+====
```

Comment:
Ingress can be disabled resp. replaced with Routes, so technically it still works for OCP3

Reply:
Changed to a note, and added a paragraph on OCP3.11 compatibility